src/http/common/entities/sr-module.ts

```diff
@@ -81,5 +83,5 @@ export class SRModule {
   })
   lastDepositBlock: number | null;
 
-  //   exitedValidatorsCount: number;
+  // exitedValidatorsCount: number;
```

Comment:
I think all commented code can be removed due to we have git :) of course if it's not a part of code explanation.